src/overview/actions.js

```diff
@@ -22,7 +22,10 @@ export function init() {
         dispatch(refreshSearch({loadingIndicator:true}))
 
         // Track database changes, to e.g. trigger search result refresh
-        onDatabaseChange(change => dispatch(handlePouchChange({change})))
+        onDatabaseChange((change) => {
+          if (getState().options.searchIndexRebuildingStatus == 'pending') return;
```

Comment:
Don't handle database changes while search index rebuilding after import - it causes dozens and hundreds of actions that stuck you browser. Maybe we need another solution for this